news/_posts/releases/2020-04-17-prestashop-1-7-7-0-beta-release.md

```diff
@@ -0,0 +1,152 @@
+---
+layout: post
+title:  "PrestaShop 1.7.7.0 Beta Is Ready To Be Tested!"
+subtitle: "You really should download it and test it"
+date:   2020-04-17 12:00:00
+authors: [ PrestaShop ]
+image: /assets/images/2020/04/1.7.7_beta_banner.jpg
+icon: icon-leaf
```

Comment:
Is this the right icon? (see http://build.prestashop.com/prestashop-icon-font/)

Reply:
It's the same as for the 1.7.6.0 beta release note

Reply:
I was just wondering if it was the _most appropriate_ icon or if it was a random choice 😄